submissions/Levi123/arcade-game/app.js

```diff
@@ -0,0 +1,116 @@
+// Enemies our player must avoid
+const DEFAULT_PLAYER = {
+    sprite: 'images/char-boy.png',
+    positionX: 200,
+    positionY: 400,
+}
+
+const DEFAULT_ENEMY = {
+    sprite: 'images/enemy-bug.png',
+    minSpeed: 200,
+    maxSpeed: 400,
+    widthEnemy: 80,
+    heightEnemy: 60,
+    positionX: -100,
+    positionY: [50, 140, 225],
```

Comment:
As a peer developer I am tasked to add more rows and more enemies.
I'd like to this in terms or "rows" rather than pixels.
Can we make values calculable from some basic consepts like "row number", "block height" etc?
This also relates to other pixel-based numbers across the code base that are calculable from basic concepts.
As a bonus you will have more accurate positioning of characters on the field.

Reply:
> As a peer developer I am tasked to add more rows and more enemies. I'd like to this in terms or "rows" rather than pixels. Can we make values calculable from some basic consepts like "row number", "block height" etc? This also relates to other pixel-based numbers across the code base that are calculable from basic concepts. As a bonus you will have more accurate positioning of characters on the field.

can you give me your telegram contact?

Reply:
> As a peer developer I am tasked to add more rows and more enemies. I'd like to this in terms or "rows" rather than pixels. Can we make values calculable from some basic consepts like "row number", "block height" etc? This also relates to other pixel-based numbers across the code base that are calculable from basic concepts. As a bonus you will have more accurate positioning of characters on the field.

i refresh my PR and DEMO. I tried to implement line work for enemy creation instead of pixel work